### DISCUSSION TITLE
Ollama WebUI: Server Connection Error ----access it remotely from linux server

### DISCUSSION THREAD

**Discussion by ThokozaniDl** [open-webui/open-webui#686] [General]
docker run -d -p 3000:8080 -e OLLAMA_API_BASE_URL=http://my_IP adrr/api -v ollama-webui:/app/backend/data --name ollama-webui --restart always ghcr.io/ollama-webui/ollama-webui:main

![ollama error](https://github.com/ollama-webui/ollama-webui/assets/101360556/fd95604b-d79e-4491-8882-cd4281b58ec0)

**Comment by justinh-rahb** [comment]
Did you try anything in `TROUBLESHOOTING.md` before coming here? You ignored the issue template and provided no details of what you already tried or proper error logs as requested, this makes it difficult to provide specific advice.
